.zuul.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- job:
+    name: bosh-openstack-cpi-release-acceptance-test-newton
+    parent: init-test
+    description: |
+      This job validation tests of bosh-openstack-cpi-release aganist OpenStack Newton release
+    run: .zuul/playbooks/bosh-openstack-cpi-release-acceptance-test/run.yaml
+    vars:
+      global_env:
+        OS_BRANCH: stable/newton
+
+- project:
+    recheck-newton:
```

Comment:
We should think about how to config pipeline and jobs, suggest to support test against M/N/O/P/Q/master OpenStack release, because lots of public cloud still use Mitaka release, and there are two modes:
1. add all version jobs into `check` pipeline, PR and comment `recheck` will tigger all of these jobs executing, and if one of these jobs failed, whole pipeline failed, need to update PR, then `recheck`.
2. add each version job into seperated pipeline, like @animationzl do in this PR, only trigger default job that test against OpenStack master branch when PR commit, if we want to test against specific OpenStack version, should comment `recheck stable/newton`.
We have to decide which one is suitable development model for CloudFoundry community.

Reply:
sure, I suggest to take the first mode, is that ok for you? @voelzmo